deploy/ingress/nginx-ingress-values.yaml

```diff
@@ -2,18 +2,6 @@
 ## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
 ##
 controller:
-  hostNetwork: true
-
-  ## Use host ports 80 and 443
-  daemonset:
-    useHostPort: false
-
-    hostPorts:
-      http: 80
-      https: 443
-
-  kind: DaemonSet
```

Comment:
I'm skeptical of totally ditching the local setup which is closely replicating our on-prem scenario

Reply:
The whole point is to simplify quick debug test setup and keep it lightweight. But we do test collecting CoreDNS IPs from both service and ingress. Besides that exposure of a CoreDNS service should be irrelevant to controller configuration.

Reply:
Should be, but practically it's not for on-prem. We are not going to cover the most critical to us code path at the moment in terratests. LB version of coredns exposure is definitely more valuable for the community. 
We might look into an attempt to automatically test both scenarious as @somaritane proposed.